lib/javatests/com/google/android/material/slider/SliderKeyTest.java

```diff
@@ -81,23 +147,46 @@ public void testMoveThumbFocus_dPad_correctThumbHasFocus() {
 
   @Test
   public void testMoveThumbFocus_plusMinus_correctThumbHasFocus() {
+    slider.requestFocus();
+
     KeyEventBuilder plus = new KeyEventBuilder(KeyEvent.KEYCODE_PLUS);
     KeyEventBuilder minus = new KeyEventBuilder(KeyEvent.KEYCODE_MINUS);
 
     sendKeyEventLeftAndRight(plus, minus);
   }
 
   @Test
-  public void testSelectThirdThumb_clickCenterDPad_activatesThirdThumb() {
+  public void testFocusThirdThumb_clickCenterDPad_activatesThirdThumb() {
+    slider.requestFocus();
+
     slider.setFocusedThumbIndex(2);
 
-    activateFocusedThumb(slider);
+    clickDpadCenter(slider);
 
     assertThat(slider.getActiveThumbIndex()).isEqualTo(2);
   }
 
+  private void activateFocusedThumb(Slider s) {
+    clickDpadCenter(s);
+  }
+
   @Test
-  public void testSelectThirdThumb_moveRight_movesThirdThumbRight() {
+  public void testActivateThirdThumb_clickCenterDPad_deactivatesThirdThumb() {
+    slider.requestFocus();
+
+    slider.setFocusedThumbIndex(2);
+
+    activateFocusedThumb(slider);
+
+    clickDpadCenter(slider);
```

Comment:
This is a duplicate of the line above, is this intentional?

Reply:
Sure, `activateFocusedThumb` activates the thumb as established in previous test. Its implementation is irrelevant in this test's context.

`clickDpadCenter` is what we're doing and what we're testing. 

The original `SliderHelper.activateFocusedThumb` has been renamed to `clickDpadCenter` because clicking dpad is no longer idempotent. Instead, having a `activateFocusedThumb` method defined right at this spot - when we just tested its impl. - seemed OK.